Numeral Systems/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+	"math"
+	"strings"
+)
+
+func toBase10(num string, fromBase int) int {
+	var base10, c, i int
+	base36 := "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	if 2 <= fromBase && fromBase <= 36 && len(num) > 0 {
+		for _, v := range num {
+			if strings.Contains(base36[:fromBase], string(v)) {
+				if '0' <= v && v <= '9' {
+					c = int(v) - 48
```

Comment:
Please replace all numeral constants with runes, e.g. it should be `'0'` instead of 48.